src/player.cpp

```diff
@@ -993,6 +1000,24 @@ void Player::LoadDatabase() {
 	}
 }
 
+void Player::LoadFonts() {
+	Font::SetDefault(nullptr, true);
+	Font::SetDefault(nullptr, false);
```

Comment:
🤔

Reply:
The ``bool`` is for "mincho or gothic" :D. This strange API is used everywhere in Font.